.circleci/config.yml

```diff
@@ -37,13 +37,15 @@ jobs:
       - save_cache: *save_yarn_cache
       - run:
           name: Build library
-          command: yarn run build
+          command: yarn run build:umd
       - run:
           name: Test packages size
           command: yarn run test:size
       - run:
           name: Type Export
-          command: yarn run build:types
+          command: |
+            yarn run build:es
```

Comment:
this way the cjs isn't needlessly built, but could also run those two build steps together

Reply:
We don't catch if the CJS build fails anymore. Is this a problem?

Reply:
didn't think of that, I'll add it back